src/main/java/uk/gov/di/config/RelyingPartyConfig.java

```diff
@@ -55,21 +55,14 @@ public static String serviceName() {
 
     public static String oidcProviderUrl() {
         return configValue(
-                "OP_BASE_URL", "https://api.build.auth.ida.digital.cabinet-office.gov.uk");
+                "OP_BASE_URL", "https://oidc.build.account.gov.uk/");
     }
 
     public static String getCloudFoundryUri() {
         if (cfEnv.isInCf() && cfEnv.getApp().getUris().size() > 0) {
             return format("https://{0}", cfEnv.getApp().getUris().get(0));
         }
-        return "http://localhost:8081";
-    }
-
-    public static int getCloudfoundryPort() {
-        if (cfEnv.isInCf()) {
-            return 8080;
-        }
-        return 8081;
```

Comment:
I've put port number at 8080 for both for now. Is there a reason we'd want a different port number for local dev?

Reply:
what port does the frontend run on? I'd assume the clash

Reply:
Nah, let's keep them the same. I think this was a hangover when we copied the code from a stub on a different service, and that service ran locally on 8080 so we'd get a port clash running them both.